src/scripts/lib/utils.js

```diff
@@ -229,6 +229,8 @@ export const dispatchEvent = (element, type, customArgs = null) => {
   return element.dispatchEvent(event);
 };
 
+export const parseJSON = (s) => (!s || typeof JSON.parse !== 'function') ? {} : JSON.parse(s.replace(/'/g, '"'));
```

Comment:
Can the argument be more descriptive please :) also the JSON.parse shouldn't be necessary?

Reply:
PR updated, maybe now its fine, simple and it does not requires parseJSON function :)